module1/lessons/nested_collections.md

```diff
@@ -7,21 +7,21 @@ tags: ruby, hashes, data structures
 ## Learning Goals
 
 * Use multiple types of collections intermingled
-* Apply common hash and array methods
-* Describe strategies for navigating nested collections
+* Apply common hash and array methods in Ruby
+* Describe strategies for navigating nested data
 
 ## Vocabulary
 * Key & Value: a paired combination of pieces of data that exist in a hash (think dictionary)
 * Element/Item: a piece of data within an array
-* Collection: a container for 0 or more pieces of data
-* Data Structure: a data organization/storage format that allows for efficient retrieval and modification of pieces of data
 * Nested Array: an array that exists as an element within another (outer) array
 * Nested Hash: a hash that exists as either an element in an array OR the value of a key within another hash
-
+* Collection: a container for 0 or more pieces of data
+* Data Structure: a data organization/storage format that allows for efficient retrieval and modification of pieces of data
+* Object: a JavaScript data structure which pairs keys & values, similar to a Ruby hash
 
 ## WarmUp
 
-Using [mod-1-be-exercises](https://github.com/turingschool-examples/mod-1-be-exercises/tree/main/lessons/nested_collections/Examples), find answers to the following: 
+Using [Mod 1 Exercises](https://github.com/turingschool-examples/se-mod1-exercises), find answers to the following: 
```

Comment:
This just links to the main repo, top level. Are we just linking to to reference the ruby exercises? Or should they be referencing a specific lesson? Or do we want them to just try these warmups on their own?